readme.md

```diff
@@ -819,6 +820,93 @@ Type: `string`
 
 The IP address used to send the request from.
 
+### Advanced HTTPS API
+
+Note: If the request is not HTTPS these options will be ignored.
+
+##### certificateAuthority
+
+Type: `string | string[] | Buffer | Buffer[]`
+
+Override the default Certificate Authorities ([from Mozilla](https://ccadb-public.secure.force.com/mozilla/IncludedCACertificateReport))
+
+```js
+// Single CA
+got('https://example.com', {ca: fs.readFileSync('./my_ca.pem')});
+
+// Multiple CAs
+got('https://example.com', {
+	ca: [
+		fs.readFileSync('./my_ca1.pem'),
+		fs.readFileSync('./my_ca2.pem')
+	]
+});
```

Comment:
I don't think we need two examples for this. Users should know how to use arrays.